sources/packages/backend/apps/api/src/route-controllers/education-program/_tests_/e2e/education-program.aest.controller.getProgramsSummary.e2e-spec.ts

```diff
@@ -0,0 +1,775 @@
+              programStatus: savedInactiveProgram.programStatus,
+              isActive: savedInactiveProgram.isActive,
+              isExpired: savedInactiveProgram.isExpired,
+              credentialTypeToDisplay: savedInactiveProgram.credentialType,
+            },
+          ],
+          count: 2,
+        });
+    },
+  );
+
+  it(
+    "Should retrieve all the inactive education programs when the search for the programs is made and none of the program statuses are selected" +
+      " except for the inactive programs.",
+    async () => {
+      // Arrange
+      const approvedProgram = createFakeEducationProgram({
+        institution: institution,
+        user: sharedUser,
+      });
+      const pendingProgram = createFakeEducationProgram(
+        {
+          institution: institution,
+          user: sharedUser,
+        },
+        { initialValue: { programStatus: ProgramStatus.Pending } },
+      );
+      const declinedProgram = createFakeEducationProgram(
+        {
+          institution: institution,
+          user: sharedUser,
+        },
+        {
+          initialValue: {
+            programStatus: ProgramStatus.Declined,
+          },
+        },
+      );
+      const inactiveProgram = createFakeEducationProgram(
+        {
+          institution: institution,
+          user: sharedUser,
+        },
+        {
+          initialValue: {
+            isActive: false,
+            programStatus: ProgramStatus.Pending,
+          },
+        },
+      );
+      const [, , , savedInactiveProgram] = await db.educationProgram.save([
+        approvedProgram,
+        pendingProgram,
+        declinedProgram,
+        inactiveProgram,
+      ]);
+
+      const endpoint = `/aest/education-program/institution/${institution.id}/summary?page=0&pageLimit=10&programNameSearch=&locationNameSearch=&inactiveProgramSearch=true`;
+      const userToken = await getAESTToken(AESTGroups.BusinessAdministrators);
+
+      // Act/Assert
+      await request(app.getHttpServer())
+        .get(endpoint)
+        .auth(userToken, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.OK)
+        .expect({
+          results: [
+            {
+              programId: savedInactiveProgram.id,
+              programName: savedInactiveProgram.name,
+              cipCode: savedInactiveProgram.cipCode,
+              credentialType: savedInactiveProgram.credentialType,
+              totalOfferings: "0",
+              submittedDate: savedInactiveProgram.createdAt.toISOString(),
+              locationId: institutionLocation.id,
+              locationName: institutionLocation.name,
+              programStatus: savedInactiveProgram.programStatus,
+              isActive: savedInactiveProgram.isActive,
+              isExpired: savedInactiveProgram.isExpired,
+              credentialTypeToDisplay: savedInactiveProgram.credentialType,
+            },
+          ],
+          count: 1,
+        });
+    },
+  );
+
+  it(
```

Comment:
This one would be a good candidate to have included a status sort desc to be validated.

Reply:
There is a problem testing this specific scenario here. The reason being that there are two `Pending` Program Status records and they can show up in any arbitrary order resulting in the test to fail arbitrarily.

Reply:
Fixed as discussed.

Reply:
I see the E2Es now having two scenarios with the `sortOrder=ASC` but the comment was suggesting one at least with the `status sort desc`. Was there any reason not to have it included in the scenarios?